lib/diagnostic.js

```diff
@@ -7,13 +7,21 @@
 // (uppercase strings without punctuation)
 // and returns that array.
 const normalizeWords = function (/* */) {
-  /* your response here */
+  let normalizedWords = [];
+  for (let i = 0; i < words.length; i++) {
+    normalizedWords[i] = words[i].toUpperCase()
+  }
+  return normalizedWords
 }
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function () {
+  let uniqueWords = [];
+  for (let i = 0; i < words.length; i++) {
+    uniqueWords[i] = words[i].toUpperCase()
+  }
+  return uniqueWords
```

Comment:
This function looks pretty similar to your function above. What we want to do is: Check if a word has already been seen, and if it hasn't, store it in an object to check against.